xml/ay_introduction.xml

```diff
@@ -118,9 +119,9 @@
     <para>
      A regular installation of &productname; &productnumber; is
      performed in a single stage. The auto-installation process, however, is
-     divided into two stages. After the installation of the basic system the
-     system boots into the second stage where the system configuration is
-     done.
+     divided into two stages. After the installation and main configuration
+     of the basic system, it is booted into a second stage in order to perform
```

Comment:
Does this mean
"the new system then boots up and runs a second stage..."
?

Currently, it sounds like something else is booting the new installation, which seems odd.